admin/services/pin-generation.service.js

```diff
@@ -17,31 +17,12 @@ const bannedWords = [
   'dim'
 ]
 
-const fourPmToday = () => moment().startOf('day').add(16, 'hours')
```

Comment:
In the old implementation, `fourPmToday` was a function.

Reply:
@jon-shipley resolved